source/documentation/breaking-changes/function-units.liquid

```diff
@@ -0,0 +1,213 @@
+---
+title: "Breaking Change: Strict Function Units"
```

Comment:
random-with-units should redirect to function-units. It looks like the old site is doing the redirect with meta tags when you visit `/documentation/breaking-changes/random-with-units`:

```
<html> <head>
<link rel=canonical href="/documentation/breaking-changes/function-units"/>
<meta http-equiv=refresh content="0; url=/documentation/breaking-changes/function-units"/>
<meta name=robots content="noindex,follow"/>
<meta http-equiv=cache-control content=no-cache />
</head> <body> </body> </html>
```

Should we do something similar, or just add an entry to `_redirects`?

Reply:
I added it to `_redirects` for now. There are a lot of other redirects we need to confirm, so we can discuss with Natalie as part of a separate story.

https://trello.com/c/9IukUAMY